build/flutter_app.gni

```diff
@@ -71,7 +71,6 @@ template("flutter_app") {
     ]
 
     outputs = [
-      bundle_path,
```

Comment:
If you specify `bundle_path` and `snapshot_path` as `outputs`, GN generates a rule with two outputs and when you specify `depfile`, Ninja expects that there will be a rule for each output in the depfile. However, that's not the case for the depfile generated by Dart in this action, the depfile contains only one rule for the snapshot file.